module/VuFind/tests/unit-tests/src/VuFindTest/View/Helper/Root/PrintArrayHtmlTest.php

```diff
@@ -0,0 +1,199 @@
+<?php
```

Comment:
There's an associated .PrintArrayHtmlTest.php.swp above which I suspect was committed inadvertently. Do you mind deleting that?